include/GL/glui.h

```diff
@@ -442,7 +444,7 @@ class GLUIAPI GLUI_StdBitmaps
 {
 public:
     GLUI_StdBitmaps(); 
-    ~GLUI_StdBitmaps();
+    ~GLUI_StdBitmaps() = default;
 
```

Comment:
No need to convince me about C++11. We are already using that in our glui-using code so it wouldn't be a problem for us.